server/routes/api/users/userRoutes.js

```diff
@@ -1,10 +1,15 @@
 import express from 'express';
-import user from "../../../controllers/usercontroller";
-import validate from "../../../middlewares/validators/validate";
+import user from '../../../controllers/usercontroller';
+import validate from '../../../middlewares/validators/validate';
 
 const router = express.Router();
 
 router.post('/signup', validate.signupValidate, user.signUp);
-router.get('/verify/:token', validate.verificationValidation,user.accountVerification);
+router.get(
+  '/verify/:token',
+  validate.verificationValidation,
+  user.accountVerification,
+);
+router.post('/signin', validate.validateSignIn, user.signIn);
```

Comment:
@bdushimi, Actually it's @irfiacre that did the initial setup of the Validation middleware when he was working on the Sign Up API... So the rest of us just followed the format he had setup. 

I think if we're to change this, then it would affect all PRs that are not yet merged. But otherwise, I have no worries updating the function naming (if all the team agrees with it).